Sensus.Shared.iOS/Callbacks/UILocalNotifications/UILocalNotificationNotifier.cs

```diff
@@ -62,7 +48,7 @@ public override void IssueNotificationAsync(string message, string id, bool play
                         AlertBody = message,
                         TimeZone = null,  // null for UTC interpretation of FireDate
                         FireDate = DateTime.UtcNow.ToNSDate(),
-                        UserInfo = new NSDictionary(NOTIFICATION_ID_KEY, id)
```

Comment:
It was working as intended previously. The conflation I see is that we're using the new callback data interface in the notifier code. These are two separate things.